hack/jenkins/e2e.sh

```diff
@@ -665,6 +665,120 @@ case ${JOB_NAME} in
           )"}
     ;;
 
+  # kubernetes-upgrade-gke-stable-latest
+  #
+  # This suite:
+  #
+  # 1. launches a cluster at release/stable,
+  # 2. upgrades the master to release/latest,
+  # 3. runs release/stable e2es,
+  # 4. upgrades the rest of the cluster,
+  # 5. runs release/stable e2es again, then
+  # 6. runs release/latest e2es and tears down the cluster.
+
+  kubernetes-upgrade-stable-latest-gke-step1-deploy)
+    : ${DOGFOOD_GCLOUD:="true"}
+    : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
+    : ${E2E_CLUSTER_NAME:="gke-upgrade-stable-latest"}
+    : ${E2E_NETWORK:="gke-upgrade-stable-latest"}
+    : ${JENKINS_PUBLISHED_VERSION:="release/stable"}
```

Comment:
No, `release/stable` is 1.0.6; `release/latest` is 1.0.7, (until the rollout is done, at which point they'll both be 1.0.7).